### ISSUE TITLE
question about AffinGrid class?

### ISSUE THREAD

**Issue by shuanggou** [Project-MONAI/MONAI#451]
I found that the output using AffineGrid class is different from the output using torch.functional.affine_grid function. The output grid of  torch.functional.affine_grid function would be normalized to [-1, 1], but I tested with AffinGrid class in monai, there was no limit to the output range, and the results were completely different.
Could you explain the difference in detail?

**Comment by Nic-Ma** [comment]
Hi @shuanggou ,

Thanks for your bug report, that's a good point.
Actually, we are adjusting the Affine transform in PR: https://github.com/Project-MONAI/MONAI/pull/444
It should already cover the normalization issue you said.

Thanks.

**Comment by wyli** [comment]
yes, the grid is currently defined in the pixel space, so that the coordinate convention is more natural (compared with the normalized ones), please see also https://github.com/pytorch/pytorch/issues/36107

**Comment by shuanggou** [comment]
> yes, the grid is currently defined in the pixel space, so that the coordinate convention is more natural (compared with the normalized ones), please see also [pytorch/pytorch#36107](https://github.com/pytorch/pytorch/issues/36107)

Thanks for your answer, so I want to know  if i use pixel space using monai, how can i do grid_sample like in torch.nn.functioncal?
